src/main/java/racing/model/RacingCar.java

```diff
@@ -0,0 +1,50 @@
+package racing.model;
+
+import java.util.Random;
+
+public class RacingCar {
+    public static final int CAN_I_MOVE = 4;
+    public static final int MAX_NAME_LENGTH = 5;
+
+    private final Random random = new Random();
+    private int progress;
+    private String name;
+
+    public RacingCar(int progress, String name) throws IllegalAccessException {
+        this.progress = progress;
+        this.name = name;
+        if (name.length() > MAX_NAME_LENGTH) {
```

Comment:
```suggestion
        if (name.length() > MAX_NAME_LENGTH) {
```

`Name` 이라는 객체가 있다면 해당 제약조건을 지키는 책임을 줄수 있을것 같아요!

Reply:
감사합니다 반영해보겠습니다😊